.ci/test_r_package.sh

```diff
@@ -82,9 +82,21 @@ export _R_CHECK_FORCE_SUGGESTS_=0
 # fails tests if either ERRORs or WARNINGs are thrown by
 # R CMD CHECK
 check_succeeded="yes"
-R CMD check ${PKG_TARBALL} \
-    --as-cran \
-|| check_succeeded="no"
+(
+    R CMD check ${PKG_TARBALL} \
+        --as-cran \
+    || check_succeeded="no"
+) &
+
+# R CMD check suppresses output, some CIs kill builds after
+# a few minutes with no output. This trick gives R CMD check more time
+#     * https://github.com/travis-ci/travis-ci/issues/4190#issuecomment-169987525
+#     * https://stackoverflow.com/a/29890106/3986677
+CHECK_PID=$!
+while kill -0 ${CHECK_PID}; do
+    echo -n -e " \b"
+    sleep 5
```

Comment:
```suggestion
    sleep 60
```

Aren't 5 seconds too frequent? With 10 minutes limit 1 minute signal seems to be quite enough.

Reply:
The polling time is the maximum amount of time the build will spend doing nothing once  `R CMD CHECK` completes.

If `R CMD CHECK` finishes running one second after a call to `sleep 60`, that is `59` seconds where the build will just be sitting doing nothing. I think 1 minute of added time is meaningful on the scale of our build times, but 5 seconds is not.

Reply:
Ah, got it! Agree with you.

Reply:
thanks! I'll merge this as soon as it builds.

Really good find with https://github.com/travis-ci/travis-ci/issues/4190 by the way, I  never would have thought of that but it's a trick I'll be able to use in other projects in the future